README.md

```diff
@@ -359,8 +359,12 @@ deprecated. To use the `classic` mode, set `CILIUM_CLI_MODE` environment variabl
 
     export CILIUM_CLI_MODE=classic
 
-> **Warning**
-> The `classic` installation mode will be removed after v0.15 release.
+> **Warnings**
+> - The `classic` installation mode will be removed after v0.15 release.
+> - Cilium CLI does not support converting `classic` mode installations to
+>   `helm` mode installations and vice versa.
+> - Cilium CLI does not support running the `helm` mode `cilium upgrade`
+>   command against classic mode installations.
```

Comment:
Should we generalize this and say that it does not support running any command (e.g., upgrade, hubble-related, clustermesh-related, ...) against classic mode installations?